pkgs/tools/networking/mailctl/default.nix

```diff
@@ -0,0 +1,85 @@
+{ mkDerivation
+, fetchFromSourcehut
```

Comment:
```suggestion
, fetchFromGitHub
```

Reply:
Why not using `fetchFromGitHub` ?

Reply:
I mean, what's the point to use urls pointint to Github in the `meta` attribute and not using the sources from Github ?

Reply:
Sorry, I meant to update the URLs in `meta` as well.  I have just fixed that.